### ISSUE TITLE
Task: Write a SOP for ACS

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-reporting#390]
Standard Operating Procedure
Advanced Cluster Security (openshift image scanning tool)

#### Describe the task

Including:
- who or which team 
- frequency
- What issues are flagged
- What resolution timeline we expect for each (critical, hich, med) severity

#### Acceptance Criteria

- [ ] An `.md` document in the docs outlining how the team uses ACS

#### Definition of Done

- [ ] AC have been met

#### Additional context

- Add any other context about the task here.
- Or here

**Comment by rdromey** [comment]
@ayeshmcg to look into this ticket - see if other teams already have some

**Comment by ayeshmcg** [comment]
@pbastia @dleard 
@joshgamache 
I asked what other teams are doing and texted in Rocket Chat channels DevOps-How To and DDS-NRM Developers. Unfortunately, I did not get any updates.
I also reached out to team members like Kuan, but they confirmed that they do not have any specific SOP in place for Advanced Cluster Security.
I have created a draft documentation for the SOP. If anyone wants to review or provide input.[Sop-ACS.pdf](https://github.com/user-attachments/files/18041440/Sop-ACS.pdf)
